frontend/src/app/tasks/layout.tsx

```diff
@@ -1,14 +1,21 @@
 'use client'
+import { useState } from 'react'
 
 import { PropsWithChildren } from 'react'
 import { Tabs, TabList, Tab, Box } from '@mui/joy'
 import Link from 'next/link'
 import { usePathname } from 'next/navigation'
+import { ContentSidebar } from '@/ui/ContentSidebar/ContentSidebar'
+import { WorkSummaryPanel } from '@/app/tasks/components/WorkSummaryPanel';
+
 
 export default function TasksLayout({ children }: PropsWithChildren) {
+  const [contentBarExpanded, setContentBarExpanded] = useState(false)
+
   const pathname = usePathname()
 
   return (
+    <>
     <Box
```

Comment:
we should add a `margin: '0 60px 0 auto'` here in to not overlap the sidebar with the content:

with the margin:

![image](https://github.com/Igalia/phpreport/assets/20492786/0a0fcc6a-a759-4c4e-ba3f-dde94677ad91)


without the margin:


![image](https://github.com/Igalia/phpreport/assets/20492786/ac1f0154-743e-4a0e-8e11-396b93c892d8)